tronbyt_server/templates/manager/firmware_form.html

```diff
@@ -22,10 +22,15 @@ <h1>{% block title %}{{ _('Generate Firmware for') }} {{ device['name'] }}{% end
     }
 </style>
 
-    <div class="left-align">
-        <label for="gen2">{{ _('Gen2?') }}</label>
-        <input type="checkbox" name="gen2" id="gen2" {% if request.form.get('gen2') %} checked {% endif %}>
-    </div>
+ <div class="left-align">
+    <label for="device_type">{{ _('Device Type') }}</label>
+    <select name="device_type" id="device_type">
+        <option value="gen1" {% if request.form.get('device_type') == 'gen1' or not request.form.get('device_type') %} selected {% endif %}>{{ _('Gen1') }}</option>
```

Comment:
How about "Tidbyt Gen1" and "Tidbyt Gen2" instead of just "Gen1" and "Gen2"? Just to be very explicit.

Reply:
I had the same thought.